<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBEditWidget</class>
 <widget class="QWidget" name="DBEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#DBEditWidget{
background-color: rgb(255, 255, 255);
}

QToolButton{
font: 22pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border:2px solid  rgb(0, 125, 165);
border-radius:15px;
background-color: rgb(0, 125, 165);
}
QLabel{
font: 14pt &quot;微软雅黑&quot;;
}

QTableWidget{
font: 14pt &quot;微软雅黑&quot;;
gridline-color: rgb(0, 0, 0);
border:1px solid rgb(0, 122, 165);
border-radius:5px;
background-color: rgb(255, 255, 255);
}
QTableWidgetItem{
outline:0px;
border:1px solid rgb(0, 0, 0);
}

QGroupBox{
font: 16pt &quot;微软雅黑&quot;;
color:rgb(0, 125, 165);
color:rgb(0,   0,   0);
border:1px none rgb(0, 125, 165);
border-radius:5px;
margin-top:20px;
}
QGroupBox:title{
subcontrol-origin: margin;
left: 5px;
}
QScrollBar:vertical{
width:20px;
background:rgb(0,0,0,0%);
margin:0px,0px,0px,0px;
padding-top:25px;   
padding-bottom:25px;    
}
QScrollBar::handle:vertical{
width:10px;
background:rgb(0,0,0,25%);
background:rgb(0,125,165,100%);
border-radius:5px;
min-height:20px;
}
QScrollBar::handle:vertical:hover{
width:10px;
background:rgb(0,125,165,80%);
border-radius:5px;
min-height:20;
}

QScrollBar:horizontal {
height:20px;
background:rgb(0,0,0,0%);
margin:0px,0px,0px,0px;
padding-left:25px;   
padding-right:25px; 
}
QScrollBar::handle:horizontal{
height:10px;
background:rgb(0,125,165,100%);
border-radius:5px;
min-width:20px;
}
QScrollBar::handle:horizontal:hover{
height:10px;
background:rgb(0,125,165,80%);
border-radius:5px;
min-width:20;
}
QToolTip{
font:9pt &quot;楷体&quot;;
color:rgb(0, 0, 0);
}
</string>
  </property>
  <widget class="QToolButton" name="tBtnImport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>505</y>
     <width>260</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>导入excel文件</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/Image/file.png</normaloff>:/Image/file.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>900</width>
     <height>490</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
spacing: 10px;
}
QCheckBox::indicator{
width:40px;
height:30px;
margin-left:20px;
image: url(:/Image/uncheck.png);
}
QCheckBox::indicator:unchecked {
image: url(:/Image/uncheck.png);
}
QCheckBox::indicator:checked {
image: url(:/Image/checked.png);
}</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnSaveDb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>505</y>
     <width>280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>保存数据库文件</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/Image/dataSearch.png</normaloff>:/Image/dataSearch.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnBack">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>505</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/Image/back.png</normaloff>:/Image/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
